### PR TITLE
Add rpower bmcreboot command for openbmc

### PR DESCRIPTION
**Pull request by xuweibj**
#3436 

Output:
```
# rpower p9euh02 bmcreboot
p9euh02 BMC: reboot
```
Output when XCATBYPASS=1
```
# XCATBYPASS=1 rpower p9euh02 bmcreboot
$VAR1 = {
          'p9euh02' => {
                         'password' => '0penBmc',
                         'bmc' => '10.6.9.254',
                         'cur_status' => 'LOGIN_REQUEST',
                         'username' => 'root'
                       }
        };

[OpenBMC development support] Using this version of xCAT, ensure firware level is at v1.99.6-0-r1, or higher.
$VAR1 = {
          'RPOWER_BMCREBOOT_REQUEST' => 'RPOWER_RESET_RESPONSE',
          'LOGIN_REQUEST' => 'LOGIN_RESPONSE',
          'LOGIN_RESPONSE' => 'RPOWER_BMCREBOOT_REQUEST'
        };

p9euh02: DEBUG POST https://10.6.9.254/login -d { "data": [ "root", "0penBmc" ] }
p9euh02: DEBUG login_response 200 OK
p9euh02: DEBUG PUT https://10.6.9.254/xyz/openbmc_project/state/bmc0/attr/RequestedBMCTransition -d {"data":"xyz.openbmc_project.State.BMC.Transition.Reboot"}
p9euh02: DEBUG rpower_reset_response 200 OK
p9euh02 BMC: reboot
monitorctrl::setNodeStatusAttributes called
```

After reboot, will can not connect to BMC using restful, and it should ping again once it comes back up
```
p9euh02: Error: 503 Service Unavailable
```

### CODE REVIEW

**Comment by whowutwut** [comment]
@xuweibj  We need some changes in the rpower pod (generated rst file) and also the Usage.pm files to reflect this new attribute.

**Comment by xuweibj** [comment]
Hi @whowutwut , I have modified, please help to review again.

File: xCAT-client/pods/man1/rpower.1.pod [hunk 1]
```diff
@@ -22,7 +22,7 @@ B<rpower> I<noderange> [B<pduon>|B<pduoff>|B<pdustat>|B<pdureset>]
 
 =head2 OpenPOWER OpenBMC:
 
-B<rpower> I<noderange> [B<off>|B<on>|B<softoff>|B<reset>|B<boot>|B<bmcstate>|B<stat>|B<state>|B<status>]
+B<rpower> I<noderange> [B<off>|B<on>|B<softoff>|B<reset>|B<boot>|B<bmcreboot>|B<bmcstate>|B<stat>|B<state>|B<status>]
```
Comment (whowutwut) [inline xCAT-client/pods/man1/rpower.1.pod]: Need another item below to describe the **bmcreboot** option.

**Comment by xuweibj** [comment]
@whowutwut Thank you for your comments. Sorry I forgot it. I have added the describe, please help to review again.

File: perl-xCAT/xCAT/Usage.pm [hunk 2]
```diff
@@ -29,10 +29,10 @@ my %usage = (
        rpower noderange [on|off|softoff|reset|boot|stat|state|status|wake|suspend [-w timeout] [-o] [-r]]
        rpower noderange [pduon|pduoff|pdustat]
      OpenPOWER BMC:
-       rpower noderange [on|off|softoff|reset|boot|bmcstate|stat|state|status]
+       rpower noderange [on|off|reset|boot|stat|state|status]
        rpower noderange [pduon|pduoff|pdustat]
      OpenPOWER OpenBMC:
-       rpower noderange [on|off|reset|boot|stat|state|status]
+       rpower noderange [on|off|softoff|reset|boot|bmcreboot|bmcstate|stat|state|status]
```
Comment (whowutwut) [inline perl-xCAT/xCAT/Usage.pm]: Thanks for fixing this for the bmcstate!

**Pull request merged**